<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homepage</class>
 <widget class="QWidget" name="homepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/img/منابع پروژه/لوگو.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);</string>
   </property>
   <property name="text">
    <string>     Todolist</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>861</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 59, 149);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>651</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Today">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>0</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);
background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
  -webkit-box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  text-shadow: 2px 1px 1px rgba(15,73,168,0.66) ;
</string>
   </property>
   <property name="text">
    <string>Today</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>263</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>356</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>449</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>542</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>470</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>....</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Dast Nevis&quot;;
color: rgb(54, 54, 54);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>your work</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>640</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>do</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_star">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>640</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:9pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>star work</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addtime">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>add time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_more1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>more information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_more2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>more information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_more3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>more information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_more4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>more information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_more5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
 
font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>more information</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);
background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
  -webkit-box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  text-shadow: 2px 1px 1px rgba(15,73,168,0.66) ;
</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/img/منابع پروژه/tasks.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>639</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_listname">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Dast Nevis&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addwork">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>670</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;

font: 8pt &quot;Dast Nevis&quot;;</string>
   </property>
   <property name="text">
    <string>add work to list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_print">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Dast Nevis&quot;;
color: rgb(1, 59, 149);
background-color: qlineargradient(spread:pad, x1:0.00480769, y1:0.501, x2:0.981, y2:0.517, stop:0.00480769 rgba(36, 128, 239, 255), stop:0.490385 rgba(69, 148, 242, 255), stop:0.697115 rgba(105, 168, 244, 255), stop:0.894231 rgba(148, 193, 248, 255), stop:1 rgba(230, 241, 254, 255));
display: inline-block;
  -webkit-box-sizing: content-box;
  -moz-box-sizing: content-box;
  box-sizing: content-box;
padding: 10px 20px;
  border: 1px solid #018dc4;
  -webkit-border-radius: 11px;
  border-radius: 11px;
  -o-text-overflow: clip;
  text-overflow: clip;
  -webkit-box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  box-shadow: 5px 5px 4px 0 rgba(0,0,0,0.2) ;
  text-shadow: 2px 1px 1px rgba(15,73,168,0.66) ;
</string>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>461</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Dast Nevis&quot;;
color: rgb(54, 54, 54);
</string>
   </property>
   <property name="placeholderText">
    <string>work description</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>lineEdit</zorder>
  <zorder>dateEdit</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_4</zorder>
  <zorder>checkBox_5</zorder>
  <zorder>checkbox_star</zorder>
  <zorder>pushButton_addtime</zorder>
  <zorder>pushButton_more1</zorder>
  <zorder>pushButton_more2</zorder>
  <zorder>pushButton_more3</zorder>
  <zorder>pushButton_more4</zorder>
  <zorder>pushButton_more5</zorder>
  <zorder>line_5</zorder>
  <zorder>label_10</zorder>
  <zorder>line_7</zorder>
  <zorder>label_listname</zorder>
  <zorder>pushButton_addwork</zorder>
  <zorder>pushButton_print</zorder>
  <zorder>pushButton_save</zorder>
  <zorder>pushButton_Today</zorder>
  <zorder>textEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
